src/services/reducers/addReducer.js

```diff
@@ -1,17 +1,29 @@
+// import { ADD } from "../constants/constants";
 
-import { ADD } from "../constants/constants";
-
-const initialTodos = { todos: [] }
+const initialTodos = {
+  todos: [
+    { text: "Finish homework", created_at: '27.06.2023', completed: false },
+    { text: "Go grocery shopping", created_at: '27.06.2023', completed: true },
+  ]
+};
 
 const addReducer = (state=initialTodos, action) => {
-  switch (action.type){
-    case ADD:
+  let newTodo;
+  switch (action.type) {
+    case "ADD":
+      newTodo = {
+        text: action.payload.text,
+        created_at: action.payload.created_at,
+        completed: false,
```

Comment:
are we using `completed` property in this PR ?

Reply:
no, was added for next steps. Removing as not related with this PR. Thanks.